clarifai/client/workflow.py

```diff
@@ -58,7 +58,7 @@ def __init__(self,
     BaseClient.__init__(self, user_id=self.user_id, app_id=self.app_id, base=base_url, pat=pat)
     Lister.__init__(self)
 
-  def predict(self, inputs: List[Input]):
+  def predict(self, inputs: List[Input], workflow_state_id: str = None):
     """Predicts the workflow based on the given inputs.
 
     Args:
```

Comment:
Can we add a description about **_workflow_state_id_** in args.

Reply:
For sure.